app/decorators/proposal_decorator.rb

```diff
@@ -9,26 +9,8 @@ def total_approvers
     object.individual_steps.count
   end
 
-  def steps_by_status
-    # Override default scope
-    object.individual_steps.with_users.reorder(
-      # http://stackoverflow.com/a/6332081/358804
-      <<-SQL
-        CASE steps.status
-        WHEN 'approved' THEN 1
-        WHEN 'actionable' THEN 2
-        ELSE 3
-        END
-      SQL
-    )
-  end
-
   def steps_in_list_order
```

Comment:
Even though we are not dropping the `flow` column, a Proposal still has a a "flow" flavor, indicated by the root step type. Would a proposal with a Parallel root step still need `steps_by_status`?